### DISCUSSION TITLE
install error

### DISCUSSION THREAD

**Discussion by gaoyan0204** [FerretDB/FerretDB#980] [General]
I want to connect feretdb with mongodb client. What is the default user name and password？

**Comment by w84thesun** [comment]
Hi, @gaoyan0204! There is no authentication enabled at the moment in FerretDB. You can connect to the database without specifying any user name and password.

**Reply by gaoyan0204** [reply]
how to set the user and password of mongodb？
In addition Can I connect ferretdb through Java program in the way of mongodb? Now the program reports an error.
com.mongodb.MongoSocketReadException: Prematurely reached end of stream。
